app/assets/stylesheets/bookstore/_general.scss

```diff
@@ -0,0 +1,347 @@
+$footer-sticky-height: 170px;
```

Comment:
You should not include all these bootstrap files like this. It is better to use gem for it. In case you want to copy them they should be in the vendors folder